Solutions/Salesforce Service Cloud/Data Connectors/SalesforceServiceCloud_API_FunctionApp.json

```diff
@@ -79,7 +79,7 @@
             "description": ">**(Optional Step)** Securely store workspace and API authorization key(s) or token(s) in Azure Key Vault. Azure Key Vault provides a secure mechanism to store and retrieve key values. [Follow these instructions](https://docs.microsoft.com/azure/app-service/app-service-key-vault-references) to use Azure Key Vault with an Azure Function App."
         },
         {
-            "description":">This data connector depends on a parser based on a Kusto Function to work as expected. [Follow these steps](https://aka.ms/sentinel-SalesforceServiceCloud-parser) to create the Kusto functions alias **SalesforceServiceCloud**"
+            "description":"**NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected which is deployed as part of the solution. To view the function code in Log Analytics, open Log Analytics/Microsoft Sentinel Logs blade, click Functions and search for the alias SalesforceServiceCloud and load the function code or click [here](https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/Salesforce%20Service%20Cloud/Parsers/SalesforceServiceCloud.txt).The function usually takes 10-15 minutes to activate after solution installation/update."
```

Comment:
Hello @v-rusraut please a whitespace between ' . ' and ' the ' 
<img width="297" alt="image" src="https://user-images.githubusercontent.com/117061676/232995671-041a550d-e8d9-4634-a6a1-9489794762e3.png">